src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -123,7 +122,6 @@ private void InitializeDelegateCommands()
         public DelegateCommand SaveImageCommand { get; set; }
         public DelegateCommand ShowSaveImageDialogAndSaveResultCommand { get; set; }
         public DelegateCommand ToggleConsoleShowingCommand { get; set; }
-        public DelegateCommand TogglePreviewBubblesShowingCommand { get; set; }
```

Comment:
Are these commands that are released in Dynamo 1.0 and 1.1.... if so they should not be removed right?